.gitignore

```diff
@@ -230,3 +230,4 @@ sklearn/feature_selection/mutual_info.py
 sklearn/feature_selection/rfe.py
 sklearn/feature_selection/univariate_selection.py
 sklearn/feature_selection/variance_threshold.py
+Pipfile
```

Comment:
Could you please remove `Pipfile` from `.gitignore`? The file is created by your virtual environment. To avoid this problem you can create the virtual environment files outside the development directory.